### PR TITLE
New refactor

### PR DESCRIPTION
**Pull request by denbit**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -43,8 +28,13 @@ class App extends Component {
 	};
 
 	async componentWillMount() {
-		await this.loadLangs();
-		LanguageCache.ua = this.state.language;
+        let lang = Cookies.get('lang');
```
Comment (denbit) [inline src/App.js]: format code please

File: src/config.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"defaultLanguage": "ua"}
```
Comment (denbit) [inline src/config.json]: why do we need two config files?

File: src/contacts.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import classNames from "classnames";
```
Comment (denbit) [inline src/contacts.js:2]: why haven't you refactored this file?

File: src/header/Lang.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
```
Comment (denbit) [inline src/header/Lang.js:3]: where is proptypes???

File: src/rotator/Arrow.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import PropTypes from "prop-types";
+import React, {Component} from 'react';
+
```
Comment (denbit) [inline src/rotator/Arrow.js:3]: rename to this.props.arrowText please

File: src/config/index.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+export const STEP = 10;
```
Comment (denbit) [inline src/config/index.js:1]: the same as above

File: src/contacts.js [hunk 7]
```diff
@@ -1,95 +1,21 @@
...
-
+import styles from "./style/ContactForm.css";
+import BigTextField from "./content/contactForm/BigTextField";
+import SmallTextField from "./content/contactForm/SmallTextField";
+import Button from "./content/contactForm/button";
 const loadText = "Loading";
-console.log(Language);
 
-function InputField(props) {
+export const InputField =(props) => {
```
Comment (denbit) [inline src/contacts.js]: why basic InputField have you left in this file?

File: src/contacts.js [hunk 8]
```diff
@@ -1,95 +1,21 @@
...
-			</div>
-		</Fragment>
-	}
-
-}
 
-const Button = (props) => {
-	return (<button onClick={props.onClick}>{props.text}</button>)
-};
+const getTranslation = (container, value) => (container && container[value]) || loadText;
```
Comment (denbit) [inline src/contacts.js]: why are you using this function?

File: src/contacts.js [hunk 9]
```diff
@@ -111,7 +37,7 @@ class ContactForm extends Component {
 		const {Consumer: Translator} = Language;
 		const withTranslation = <Translator>
 			{({contact_form}) => {
-				const sent = this.state.sent == undefined ? false : true;
+				const sent = this.state.sent === undefined ? false : true;
 				const formComponent = <Fragment>
 					<h1 className={styles.title}> {getTranslation(contact_form, 'title')}</h1>
```
Comment (denbit) [inline src/contacts.js]: the same maybe you should use 

> translate

>

File: src/content/contactForm/SmallTextField.js [hunk 10]
```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import styles from "../../style/ContactForm.css";
+import classNames from "classnames";
+import React, { Component, Fragment} from "react";
+import {InputField} from "../../contacts";
```
Comment (denbit) [inline src/content/contactForm/SmallTextField.js]: why are you separeted final implementation but left InputField in contacts.js

File: src/content/contactForm/button.js [hunk 11]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+
+const Button = (props) => {
```
Comment (denbit) [inline src/content/contactForm/button.js:4]: where is proptypes?

File: src/func.list.js [hunk 12]
```diff
@@ -1,3 +1,20 @@
...
+
+/**
+ * @param {object} container
+ * @param {string} value
+ * @returns {string}
+ *
+ * @author elika.filin@langivi.technology
+ */
+function translate(container, value) {
+	const {defaultLanguage} = config.DEFAULT_LANGUAGE;
```
Comment (denbit) [inline src/func.list.js]: would it work?

File: src/header/Lang.js [hunk 13]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from "prop-types";
+
+function Lang(props) {
+    return (<div className="lang" onClick={props.switcher}>{props.lang.toLowerCase()}</div>);
```
Comment (denbit) [inline src/header/Lang.js:5]: why have you missed lang prop?

**Review by denbit — COMMENTED** [review]
please check comments

File: src/contacts.js (hunk 7 above)
Comment (ElikaFilin) [reply src/contacts.js]: done

File: src/contacts.js (hunk 8 above)
Comment (ElikaFilin) [reply src/contacts.js]: What do you mean?

File: src/content/contactForm/SmallTextField.js (hunk 10 above)
Comment (ElikaFilin) [reply src/content/contactForm/SmallTextField.js]: done

File: src/content/contactForm/button.js (hunk 11 above)
Comment (ElikaFilin) [reply src/content/contactForm/button.js:4]: done

File: src/func.list.js (hunk 12 above)
Comment (ElikaFilin) [reply src/func.list.js]: done

File: src/header/Lang.js (hunk 13 above)
Comment (ElikaFilin) [reply src/header/Lang.js:5]: done

**Pull request merged**